docs/products/cassandra/howto/enable-cross-cluster-replication.rst

```diff
@@ -0,0 +1,174 @@
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select **+Create service**.
+3. In the **Create service** view
+
+   1. Select Apache Cassandra® and, next, a cloud provider and a cloud region for your service.
+   2. To activate CCR, select the **Enable Cross Cluster Replication** toggle, which unfolds an additional section for setting up another Apache Cassandra service.
+   3. Select a cloud provider and a cloud region for the second service.
+   4. Select one service plan for your two new services.
+   5. Optionally, add an additional disk storage for your services.
+   6. Enter names for your services.
+
+4. Verify your choices in the **Services summary** card on the right and, if your configuration turns out as expected, select **+Create services** if the setting looks.
+
+.. topic:: Result
+   
+   You've created two new services that are connected for CCR purposes. You can preview CCR details for your services in the **Overview** tab under **Cross Cluster replication**.
+
+On an existing service
+''''''''''''''''''''''
+
+1. Log in to the `Aiven console <https://console.aiven.io/>`_.
+2. From the **Current services** view, select an Aiven for Apache Cassandra service on which you'd like to enable CCR.
+3. In the **Overview** tab of your service's page, navigate to **Cross Cluster Replication** and select **Migrate to Cross Cluster**.
+
+   .. note::
+      
+      When you enable CCR on a particular service, you connect this service to a new service that comes with the same plan, number of nodes, and disk storage.
+
+4. In the **Create a Cross Cluster Replica** view
+   
+   1. Select a cloud provider and a cloud region, define a name for your new service, and select **Continue**.
+   2. Examine the **Service Summary** section and make sure the configuration for your new service meets your expectations. If so, select **Create Cross Cluster Replica**. 
+
+.. topic:: Result
+   
+   CCR has been enabled by connecting your service to another new service, which is now visible in the **Overview** tab under **Cross Cluster replication**.
+
+Enable CCR with CLI
+-------------------
+
+You can enable CCR for your new or existing Aiven for Apache Cassandra service using the Aiven CLI.
+
+.. tip::
+
+   Check out how to get started with the Aiven CLI in :doc:`Aiven CLI </docs/tools/cli>`.
+
+On a new service
+''''''''''''''''
+
+Use the :ref:`avn service create <avn-cli-service-create>` command to create a new service and add the ``service_to_join_with`` parameter. The parameter indicates a service to which you'd like to connect to enable CCR and create a CCR service pair.
+
+.. code-block:: bash
+
+   avn service create                                   \
+      --service-type cassandra                          \
+      --cloud cloud_region_name                         \
+      --plan service_plan_name                          \
+      -c cassandra.datacenter=datacenter_name           \
```

Comment:
also the text above misses the fact that the datacenter needs to be different. Maybe add a common section that explains the contents of service_to_join_with and cassandra.datacenter and link to it from all the recipes? in order not to repeat it again and again